bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/AutoRecoveryMainTest.java

```diff
@@ -49,11 +52,9 @@ public void testStartup() throws Exception {
         AutoRecoveryMain main = new AutoRecoveryMain(confByIndex(0));
         try {
             main.start();
-            Thread.sleep(500);
-            assertTrue("AuditorElector should be running",
-                    main.auditorElector.isRunning());
-            assertTrue("Replication worker should be running",
-                    main.replicationWorker.isRunning());
+            await().atMost(1, SECONDS).untilAsserted(() ->
```

Comment:
Remove the atMost logic and use the default most await time?

Reply:
The default await time is 10s